src/transactions/0_send.js

```diff
@@ -46,8 +46,8 @@ export default function send({
 	const fee = data ? SEND_FEE + DATA_FEE : SEND_FEE;
```

Comment:
@Isabello Should we still be doing maths with fees as vanilla numbers? If not it would make more sense to have the fee constants themselves as strings.

Reply:
I would prefer keeping internal (Lisk-JS) fee in numbers for the calculations. As soon as we put it in a transaction object we can make sure it is a string.